client/dashboard/sites/settings-plan/summary.tsx

```diff
@@ -0,0 +1,44 @@
+import { siteCurrentPlanQuery, sitePurchaseQuery } from '@automattic/api-queries';
+import { useQuery } from '@tanstack/react-query';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { payment } from '@wordpress/icons';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { getSitePlanDisplayName, useSitePlanManageURL } from '../../utils/site-plan';
+import { isRelativeUrl } from '../../utils/url';
+import type { Site } from '@automattic/api-core';
+import type { Density } from '@automattic/components/src/summary-button/types';
+
+export default function SettingsPlanSummary( {
+	site,
+	density,
+}: {
+	site: Site;
+	density?: Density;
+} ) {
+	const { data: plan, isLoading: isLoadingPlan } = useQuery( siteCurrentPlanQuery( site.ID ) );
+	const { data: purchase, isLoading: isLoadingPurchase } = useQuery( {
+		...sitePurchaseQuery( site.ID, parseInt( plan?.id ?? '' ) ),
+		enabled: !! plan?.id,
+	} );
+
+	const url = useSitePlanManageURL( site, purchase );
+	if ( ! url || ( url && ! isRelativeUrl( url ) ) ) {
```

Comment:
Nit: I think it can be simply like this due to short-circuiting

```suggestion
	if ( ! url || ! isRelativeUrl( url ) ) {
```

Reply:
Ah, of course 🤦🏻‍♂️
I thought I saw TS not being happy about but turned out I had the wrong conditional back then.